pages/app/app-context.tsx

```diff
@@ -28,6 +29,7 @@ const appContextDefaults: AppContextType = {
     density: Density.Comfortable,
     visualRefresh: THEME === 'default',
     motionDisabled: false,
+    analyticsEnabled: false,
```

Comment:
Do we really need this conditionality? Can we keep it always on in our development (bleeding edge mode on)?